tests/Runtime/LambdaRuntimeTest.php

```diff
@@ -81,6 +81,19 @@ public function test exceptions in the handler result in an invocation 
         $this->assertErrorInLogs('RuntimeException', 'This is an exception');
     }
 
+    public function test nested exceptions in the handler result in an invocation error()
+    {
+        $this->givenAnEvent(['Hello' => 'world!']);
+
+        $this->runtime->processNextEvent(function () {
+            throw new \RuntimeException('This is an exception', 0, new \Exception('The original exception.'));
+        });
+
+        $this->assertInvocationErrorResult('Exception', 'The original exception.');
```

Comment:
We should probably report the wrapping exception (i.e. the one that was last trown), because it's the exception actually thrown by the app.

Reply:
Yes you're right